health/notifications/custom/metadata.yaml

```diff
@@ -0,0 +1,165 @@
+# yamllint disable rule:line-length
+---
+- id: 'notify-custom'
+  meta:
+    name: 'Custom'
+    link: ''
+    categories:
+      - notify.agent
+    icon_filename: 'custom.png'
+  keywords:
+    - custom
+  overview:
+    notification_description: |
+      Netdata Agent's alert notification feature allows you to send custom notifications to any endpoint you choose.
+    notification_limitations: ''
+  setup:
+    prerequisites:
+      list:
+        - title: ''
+          description: |
+            - Access to the terminal where Netdata Agent is running
+    configuration:
+      file:
+        name: 'health_alarm_notify.conf'
+      options:
+        description: 'The following options can be defined for this notification'
+        folding:
+          title: 'Config Options'
+          enabled: true
+        list:
+          - name: 'SEND_CUSTOM'
+            default_value: ''
```

Comment:
```suggestion
            default_value: 'YES'
```

Reply:
Is the default value of this really YES? @MrZammler

Reply:
Actually, most of them (SEND_) are, i.e. https://github.com/netdata/netdata/blob/953d0a0b620c7bdb7c6820b367a2893360a1063b/health/notifications/health_alarm_notify.conf#L874

https://github.com/netdata/netdata/blob/953d0a0b620c7bdb7c6820b367a2893360a1063b/health/notifications/health_alarm_notify.conf#L810

I'm checking a default `./edit-config health_alarm_notify.conf` on a new agent too. The fact that they don't actually send something is because the other settings (i.e. RECIPIENT) is empty.